strapi/strapi
# Add collapse option to dynamic zone

### What does it do?

- Adds an options to the Dynamic Zone to have a collapse by default option

### Why is it needed?

- Lots of content inside of components and lots of components inside a dynamic zone makes it difficult to add/move components. Many discussions been raised about this

https://github.com/strapi/strapi/issues/16737
https://forum.strapi.io/t/strapi-v4-dynamic-zono-components-to-be-closed-by-default/25661
https://github.com/strapi/strapi/issues/15655

### How to test it?

- Add a dynamic zone to a content type
- Go into Advanced Settings
- Select the new option for Collapsed by Default
- Add a component field
- Create a new entry
- Add a couple of the new components
- Save the page
- Refresh and confirm that the items are collapsed by default

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

https://github.com/strapi/strapi/issues/16737
https://github.com/strapi/strapi/issues/15655
